### ISSUE TITLE
Consider starting a list of words with varying meanings

### ISSUE THREAD

**Issue by sebbASF** [inclusivenaming/org#79]
There are some words in common usage that have meanings which vary between different variants of English.

For example:
- "table", as in "the motion was tabled": in the US this means "postponed"; in the UK it means "put to the meeting"

Whilst these words are unlikely to be harmful, they can be considered exclusionary.

There are also words which are ambiguous in any English variant.
Whilst these are not exclusionary of themselves, they are best avoided.
For example:
- "bi-weekly" can mean either "every other week" or "twice a week"

It might be worth maintaining a separate list of such words.

**Comment by sebbASF** [comment]
Also
spring, summer, autumn(fall) and winter: these refer to different months in different hemispheres.

**Comment by markcmiller86** [comment]
I recommend this be closed as out of scope.

@sebbASF, please have look at the [top bullet here](https://inclusivenaming.org/about/) and consider known examples such as "master/slave" and "blacklist/whitelist" to help set your thinking cap for scope.

**Comment by richsalz** [comment]
Strongly agree with @markcmiller86; close this as out of scope.

**Comment by sebbASF** [comment]
Does the language have to be *both* harmful *and* exclusionary?

I agree these words are not harmful, but they can effectively exclude people from different cultures.
My intention was to collect a list of such words that people may not realise can cause issues for cultures other than their own.

**Comment by nicorikken** [comment]
@sebbASF I agree that sticking to plain and simple English is important, and terms like bi-weekly or even fortnight (I heard that in some communities) can require a dictionary to be sure about the meaning. In that sense it can be equated to the use of local timezones that require a translation. Although I do consider complicated language to be 'exclusionary', and having a list would be nice for checking it with tools, it might not have the highest priority for INI I guess?

**Comment by sebbASF** [comment]
I am not suggesting that it is of the same priority as words that are harmful.
However, I think it is still worth providing the information so authors can be aware of the issue.

Note: I had assumed that fortnight was universally understood; it seems not, so that should go on the list.

As to bi-weekly: that has two different meanings.
A dictionary cannot determine which is intended.

**Comment by sebbASF** [comment]
Thanks, but it looks like the Plain Language site is US-specific.
